iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -213,7 +219,9 @@ public void commitDropTable(org.apache.hadoop.hive.metastore.api.Table hmsTable,
   @Override
   public void preAlterTable(org.apache.hadoop.hive.metastore.api.Table hmsTable, EnvironmentContext context)
       throws MetaException {
-    super.preAlterTable(hmsTable, context);
+    if (!isSupportedAlterOperation(context)) {
+      super.preAlterTable(hmsTable, context);
```

Comment:
Since we've added ADDPROPS and DROPPROPS to our allowed list as well, maybe we can remove the super call? It seems like it wouldn't add any additional value at this point?